docs/modules/examples/pages/knowledge-graph.adoc

```diff
@@ -0,0 +1,288 @@
+= Knowledge Graph
+
+image::https://colab.research.google.com/assets/colab-badge.svg[align="left",link="https://colab.research.google.com/github/datastax-labs/knowledge-graphs-langchain/blob/main/notebook.ipynb"]
+
+Use RAGStack, https://python.langchain.com/docs/use_cases/graph/constructing/#llm-graph-transformer[`LLMGraphTransformer`], and https://www.datastax.com/products/datastax-astra[DataStax AstraDB] to extract knowledge triples and store them in a vector database.
+
+== Prerequisites
+
+* An active https://www.datastax.com/products/datastax-astra[DataStax AstraDB]
+* Python 3.11 (to use `Union` and `self` hints)
+* OpenAI API key
+
+== Environment
+
+. Install dependencies:
++
+[source,bash]
+----
+pip install ragstack-ai-knowledge-graph python-dotenv
+----
++
+. Create a `.env` file and store the necessary credentials.
++
+[source,bash]
+----
+OPENAI_API_KEY="sk-..."
+ASTRA_DB_DATABASE_ID="670d40c2-80f9-4cb0-8c74-d524dd6944d1"
```

Comment:
Do we really want a database ID showing up? Is this a common pattern?

Reply:
It's fake